pymc3/distributions/multivariate.py

```diff
@@ -13,7 +13,7 @@
 
 import pymc3 as pm
 
-from pymc3.math import logdet, tround
```

Comment:
`logdet` is still used in `log_normal_mv` in `dist_math.py`, which is in turn used in `approximations.py`, so perhaps it would be better to leave this to another PR.